code/modules/reagents/reagents/random.dm

```diff
@@ -0,0 +1,158 @@
+
+
+// subtypes of stuff in here will be avoided when randomizing interactions.
```

Comment:
Is this file included?

Reply:
Hmm, indeed it's not, it must have been added when I grabbed the PR from Eclipse.

I'll remove it. random_effects.dm as well

Reply:
Done